aiodocker/jsonstream.py

```diff
@@ -2,7 +2,12 @@
 import json
 import logging
 
-import aiohttp
+# aiohttp has no errors.py
+try:
+    import aiohttp.errors as errors
+except:
+    import aiohttp.client_exceptions as errors
+
```

Comment:
aiohttp recommends to use "direct" references to publicly exported names from `aiohttp`, not `aiohttp.errors` or whatever submodules, as the internal module structure may change in future versions.
This diff looks like reverting my changes in this sense.